test/python/test_fill.py

```diff
@@ -0,0 +1,75 @@
+# ==============================================================================
+#  Copyright 2018 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+"""nGraph TensorFlow bridge fill operation test
+
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+import pytest
+
+from common import NgraphTest
+
+
+class TestFillOperations(NgraphTest):
+  @pytest.mark.parametrize(
+      ("shape", "value"),
+      ((
+       ([1, 2, 3, 1], 10),
```

Comment:
Might also try shapes of `[]` and `[0]`, if those happen to give reasonable results in the non-nGraph impelmentation (I don't know if they actually do).